src/backend/distributed/executor/adaptive_executor.c

```diff
@@ -846,6 +906,12 @@ CreateDistributedExecution(RowModifyLevel modLevel, List *taskList, bool hasRetu
 	execution->connectionSetChanged = false;
 	execution->waitFlagsChanged = false;
 
+	execution->useRemoteTransactionBlocks = REMOTE_TRANSACTION_BLOCKS_ALLOWED;
+	if (isOutsideTransaction)
```

Comment:
suggestion: replace `isOutsideTransaction ` with `excludeFromTransaction`, in the sense that the caller chooses to exclude the execution from the transaction block.

`isOutsideTransaction` feels a bit vague to me.

Reply:
I think that fits more into what we are doing, thanks for the suggestion